### ISSUE TITLE
Tablet portrait adjustments

### ISSUE THREAD

**Issue by marcelaquint** [Aerochimps/aerochimps-dot-com#23]
I've only designed the wireframe cause i don't think we'll need the graphic project. @idmarjr let me know if you need something.

![aerochimps ipad portrait](https://f.cloud.github.com/assets/1020108/2466802/0b538e96-afb5-11e3-874c-b9e518e34c8f.png)
- The menu is not hidden anymore, it's centered despite the logo, the font-size is something around 28px and it has 30px as margin-right;
- The content size is centtered and has a 500px max-width;
- Basically you'll start setting every image (except the teams images), text and line height more 25% (and then we will make more adjustments if necessary);
- I'm not sure about the margins, but I guess we can double every margin between section/content. If you had 15px to mobile, now you have 30px. Let's try and see how it works. :)
